ui/client/components/graph/TypeSuggest.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import PropTypes from "prop-types";
+import AceEditor from 'react-ace';
+import _ from "lodash";
+import ProcessUtils from "../../common/ProcessUtils";
+import ActionsUtils from "../../actions/ActionsUtils";
+import {connect} from 'react-redux'
+
+import 'brace/mode/jsx';
+
+import 'brace/ext/searchbox';
+import '../../brace/mode/spel'
+import '../../brace/mode/sql'
+import '../../brace/theme/nussknacker'
+import ValidationLabels from "../modals/ValidationLabels";
+import {allValid} from "../../common/Validators";
+import TypeSuggester from "./TypeSuggester";
+import {v4 as uuid4} from "uuid";
+
+class TypeSuggest extends React.Component {
```

Comment:
Currently I can write arbitrary text in input, and it will be accepted, but then it will be converted to null underneath
and I'll know about it only after Save and some cryptic exception...